### DISCUSSION TITLE
Create user with same email address

### DISCUSSION THREAD

**Discussion by IPeter1982** [OrchardCMS/OrchardCore#12921] [Q&A] [answered]
I want to create multiple Orchard users with different username but same email address.
I've configured the IdentityOptions to RequireUniqueEmail = false, but when I try to create a second user the same email address I get an error: "_Email 'email@email.com' is already used_." 
Program.cs:
`services.Configure<IdentityOptions>(options =>
{
    options.Password.RequireDigit = false;
    options.Password.RequireLowercase = true;
    options.Password.RequireUppercase = true;
    options.Password.RequireNonAlphanumeric = false;
    options.Password.RequiredUniqueChars = 5;
    options.Password.RequiredLength = 10;
    options.User.RequireUniqueEmail = false;
});`

The password options are working, so I don't know why the RequiredUniqueEmail is not.
Any ideas?

**Comment by hishamco** [comment, marked as answer]
FYI we are using email as a username to identify the logged in use,i may be we can log a warning message to avoid set this option

**Comment by hishamco** [comment]
FYI we are using email as a username to identify the logged in use,i may be we can log a warning message to avoid set this option
